### ISSUE TITLE
TypeError: d is not a function when run faceapi detect face

### ISSUE THREAD

**Issue by Ogafais** [justadudewhohacks/face-api.js#875]
Hi there, i bump into this issue whenever i run this code const detections = await faceapi.detectAllFaces(img).withFaceLandmarks().withFaceDescriptors(); it rerturn me TypeError: d is not a function. Thank you in advance

**Comment by faustinopsy** [comment]
if you post part of your code, help may be faster.

**Comment by Ogafais** [comment]
Sorry, here my code. FYI i did try to feed the faceapi both using video and image from my asset folder and still receive the same result :(

this.webcamElement = document.getElementById('webcam');
const detections = await faceapi.detectSingleFace(this.webcamElement ); //feed my webcam into faceapi

**Comment by vladmandic** [comment]
@Ogafais full code please. this looks like something in tfjs init sequence error, so its important to know how all libraries were loaded.

**Comment by tonyfverdu** [comment]
That is incorrect code:  const detecciones = awaitr faceapi.detectSingleFace(this.webcamElement);

In documentation of API:
const detection = await faceapi.detectSingleFace(input)  //  input must be a "image" or "video", not an element HTML
